### ISSUE TITLE
SSR Compatible 

### ISSUE THREAD

**Issue by samuelcastro** [xiaolin/react-image-gallery#576]
Hi, is this library SSR compatible? (I'm using next.js)

Also, is it possible to overwrite the `img` component to use a custom one like, [next.js Image ](https://nextjs.org/docs/api-reference/next/image) component?

Thank you!

**Comment by xiaolin** [comment]
Yes, it should be SSR friendly. you can use custom render function to replace with next.js image component. see `renderItem` in props.

**Comment by mattdell** [comment]
Just to +1...

We've been using this in Next.js for nearly a year, not yet with the Image component as we don't host our gallery images from our src.

**Comment by xiaolin** [comment]
closing for now, feel free to re-open if there are issues.
